### PR TITLE
Added pr templates

### PR DESCRIPTION
**Pull request by AlessandroMinoccheri**

### CODE REVIEW

**Comment by AlessandroMinoccheri** [comment]
Do you mean in both templates @garak ?

**Comment by garak** [comment]
> Do you mean in both templates @garak ?

Yes, I guess that it makes sense for both

File: .github/PULL_REQUEST_TEMPLATE/Improvement.md [hunk 1]
```diff
@@ -12,6 +12,7 @@ about: You have some improvement to make badge-poser better? 🎁
 | New Feature | yes
 | RFC         | yes/no
 | BC Break    | yes/no
+| Issue       | Fix #...
```
Comment (JellyBellyDev) [inline .github/PULL_REQUEST_TEMPLATE/Improvement.md:15]: `Close`?

Comment (AlessandroMinoccheri) [reply .github/PULL_REQUEST_TEMPLATE/Improvement.md]: I was unsure which to use

Comment (garak) [reply .github/PULL_REQUEST_TEMPLATE/Improvement.md]: it should be the same

Comment (AlessandroMinoccheri) [reply .github/PULL_REQUEST_TEMPLATE/Improvement.md]: Yeah, I have seen both solutions but for me `Close` is more explicit.

**Pull request merged**